modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -640,7 +654,8 @@ public CompletableFuture<Ignite> start(NodeBootstrapConfiguration cfg) {
                                     indexManager,
                                     qryEngine,
                                     clientHandlerModule,
-                                    (IgniteComponent) deploymentManager
+                                    (IgniteComponent) deploymentManager,
+                                    distributedConfigurationUpdater
```

Comment:
This is not correct: `distributedConfigurationUpdater` must be started before `cmgMgr`